kml.cc

```diff
@@ -1496,6 +1496,10 @@ void KmlFormat::kml_mt_simple_array(const route_head* header,
       writer->writeTextElement(QStringLiteral("gx:value"), wpt->temperature_has_value()?
                                QString::number(wpt->temperature_value(), 'f', 1) : QString());
       break;
+    case wp_field::sat:
+      writer->writeTextElement(QStringLiteral("gx:value"), wpt->sat?
+                                QString::number(wpt->sat, 'f', 1) : QString());
```

Comment:
If we do this you should use QString::number(wpt->sat).  Don't convert implicitly convert it to a double.